core/src/test/java/bio/terra/pearl/core/service/kit/StubPepperDSMClientTest.java

```diff
@@ -0,0 +1,83 @@
+package bio.terra.pearl.core.service.kit;
+
+import bio.terra.pearl.core.BaseSpringBootTest;
+import bio.terra.pearl.core.factory.EnvironmentFactory;
+import bio.terra.pearl.core.factory.StudyEnvironmentFactory;
+import bio.terra.pearl.core.factory.StudyFactory;
+import bio.terra.pearl.core.factory.kit.KitRequestFactory;
+import bio.terra.pearl.core.factory.participant.EnrolleeFactory;
+import bio.terra.pearl.core.model.EnvironmentName;
+import bio.terra.pearl.core.model.study.StudyEnvironmentConfig;
+import bio.terra.pearl.core.service.study.StudyEnvironmentService;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.transaction.annotation.Transactional;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasProperty;
+
+class StubPepperDSMClientTest extends BaseSpringBootTest {
```

Comment:
Might seem like overkill to have a test for a stub implementation, but it was a stepping stone to updating `PepperDSMClient.fetchKitStatusByStudy` to take a study name instead of a UUID.